tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/e2e-upgrade-test.go

```diff
@@ -102,15 +66,58 @@ var _ = Describe(
 				_, err = newIbguBuilder.WaitUntilComplete(30 * time.Minute)
 				Expect(err).NotTo(HaveOccurred(), "Prep and Upgrade IBGU did not complete in time.")
 
+				upgradeSuccessful = true
```

Comment:
I think it's best to be plugged after row 75, which really indicated a success where we are able to pull the cluster info. Also we might actually need the extracted cluster info in order to run other tests?